spread.yaml

```diff
@@ -97,9 +102,12 @@ exclude:
     - .git
 
 prepare-each: |
-    journalctl --rotate
+    # systemd on 14.04 does not know about --rotate
+    # or --vacuum-time. Gracefully handle both commands
+    # failing.
```

Comment:
We don't want to ignore errors on those. We can do this instead:
```
if [ "$SPREAD_SYSTEM" = ubuntu-14.04-64 ]; then
...
else
...
fi
```

Then, we need to find something to put on that if clause for 14.04 rather than just disabling it, because otherwise the content of `debug-each` below will cause a ridiculous amount of unrelated logs to be shown on errors, to the point of making spread non-useful.

There must be a way to clean logs on 14.04, right?